Site/BSSwFellowshipProgram/People/KatzDa.md

```diff
@@ -18,7 +18,7 @@
 ### Measure impact, give attribution, and recognize productivity with scientific software citation.
 BSSw Fellow Daniel S. Katz developed principles for software citation to make scientific software more sustainable by providing credit to its developers. Research paper citations are tracked to measure impact; similarly, software citations should be tracked to measure impact and recognize the significant time spent developing software and data. He led working groups to produce the principles, and he is currently working with publishers on implementation in institutionalized citation practices. Learn more about software citation through his webinar recording and slides: <a href="https://ideas-productivity.org/events/hpc-best-practices-webinars/#webinar017"> Software Citation Today and Tomorrow</a>, April 2018.
 
-Daniel is a Research Associate Professor, Computer Science, Electrical and Computer Engineering, and the School of Information Sciences at The University of Illinois at Urbana-Champaign and Assistant Director for Scientific Software and Applications at the National Center for Supercomputing Applications.
+Daniel is Chief Scientist at the National Center for Supercomputing Applications, University of Illinois at Urbana-Champaig, and a Research Associate Professor in Computer Science, Electrical and Computer Engineering, and the School of Information Sciences.
```

Comment:
For whoever merges this PR...there is an `n` missing at end of `Urbana-Champaig`

Reply:
Thanks - I fixed it...